### ISSUE TITLE
headers in preflight response

### ISSUE THREAD

**Issue by jiangen1016** [launchdarkly/js-client#130]
When I use launchdarkly on my localhost with Angular 6, that is work, but sometimes will tell me `net::ERR_INCOMPLETE_CHUNKED_ENCODING 200 (OK)`. what is this mean?
on my online project, they will tell me error like this 
`Access to fetch at 'https://clientstream.launchdarkly.com/eval/****' from origin 'https://****' has been blocked by CORS policy: Request header field cache-control is not allowed by Access-Control-Allow-Headers in preflight response.
leads:1 GET https://clientstream.launchdarkly.com/eval/**** net::ERR_ABORTED 504 (Gateway Timeout)`
What can I do ? Or my network problem？Can I set Request for launchdarkly? Thanks!!!

**Comment by eli-darkly** [comment]
Thanks for your interest. This sounds like a support ticket that should be directed to our support team here: https://support.launchdarkly.com/hc/en-us/requests/new - they can investigate it and consult with the SDK developers if necessary.

**Comment by gavyaggarwal** [comment]
I am also getting this issue on version 2.10.2. Looking at the launch darkly source code, it seems like this should be fixed in 2.15.2 with the `skipDefaultHeaders` flag. Is there an ETA for this release?

**Comment by eli-darkly** [comment]
@gavyaggarwal I'm not sure what you mean by "an ETA for this release": 2.15.2 was released on 2019-11-15. Also, the `skipDefaultHeaders` property you're referring to is not part of the JS SDK's API; it is part of the [`js-eventsource`](https://github.com/launchdarkly/js-eventsource) polyfill, and the setting of `skipDefaultHeaders` in the JS SDK code (which has been there since 2.12.4) simply enables that behavior _if_ you are using that polyfill; other polyfills, and the native EventSource implementation, do not have that option.

Normally, native browser implementations of EventSource do not have this problem. If you are using a polyfill, I would recommend trying the most recent version of `js-eventsource`. However, I have also seen reports of a [similar problem](https://stackoverflow.com/questions/51659368/how-to-disable-preflight-cache-control-headers-for-angular-service-worker-when) that apparently is not due to EventSource itself but something more specific to Angular... so I would not assume that all similar symptoms have the same cause. Again, I would recommend contacting support to look at your specific circumstances.
